### PR TITLE
Addition of AniDB

### PR DESCRIPTION
**Pull request by hugonun**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [X] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [X] Your additions are ordered alphabetically
- [X] Your submission has a useful description
- [X] The description does not end with punctuation
- [X] Each table column should be padded with one space on either side
- [X] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [X] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ API | Description | Auth | HTTPS | CORS |
 ### Anime
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [AniDB](https://wiki.anidb.net/API) | Anime Database | No | Yes | Unknown |
```
Comment (yannbertrand) [inline README.md:80]:
```suggestion
| [AniDB](https://wiki.anidb.net/HTTP_API_Definition) | Anime Database | `apiKey` | Yes | Unknown |
```

**Review by yannbertrand — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hugonun) [reply README.md:80]: I agree with the link change, but the authentication part is a little different. It is not a key, but a "client", an user-defined string that is required to be registered with an account, but handled like an user-agent.

Example: http://www.anidb.net:9001/httpapi?request=anime&client=animerank&clientver=1&protover=1&aid=1

I also just noticed that the API doesn't support HTTPS, my bad!

Shall we keep the authentication part with a `No`, have it be `apiKey` or is there another option?

Comment (yannbertrand) [reply README.md:80]: `apiKey` would be more appropriate as you need an account

Comment (yannbertrand) [reply README.md:80]:
```suggestion
| [AniDB](https://wiki.anidb.net/HTTP_API_Definition) | Anime Database | `apiKey` | No | Unknown |
```

**Comment by hugonun** [comment]
Suggested change committed, sorry about the delay!

**Review by matheusfelipeog — APPROVED** [review]
Thanks for your contribution @hugonun ;)

**Pull request merged**